root.tsup.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'tsup';
+import { esbuildPluginFilePathExtensions } from 'esbuild-plugin-file-path-extensions';
+import { copy } from 'esbuild-plugin-copy';
+
+export default defineConfig([
+    {
+        entry: ['src/**/*.ts'],
+        format: ['cjs', 'esm'],
+        target: 'esnext',
+        outDir: 'dist',
+        dts: true,
+        sourcemap: true,
+        clean: true,
+        tsconfig: './tsconfig.build.json',
+        esbuildPlugins: [
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+            esbuildPluginFilePathExtensions({ esmExtension: 'js' }),
+            copy({
+                resolveFrom: 'cwd',
+                assets: {
+                    from: ['./src/**/*.json'],
```

Comment:
Does this retain the path to the `src` folder, or does it dump all `json` files at the root of `dist`?

Does `tsup` not have built-in support for including static files?